ui/espresso/IntentsAdvancedSample/app/src/androidTest/java/com/example/android/testing/espresso/intents/AdvancedSample/ImageViewHasDrawableMatcher.java

```diff
@@ -16,26 +16,27 @@
 
 package com.example.android.testing.espresso.intents.AdvancedSample;
 
-import androidx.test.espresso.matcher.BoundedMatcher;
+import androidx.test.espresso.matcher.BoundedDiagnosingMatcher;
 import android.view.View;
 import android.widget.ImageView;
 
 import org.hamcrest.Description;
+import org.hamcrest.Matcher;
 
 /**
  * A Matcher for Espresso that checks if an ImageView has a drawable applied to it.
  */
 public class ImageViewHasDrawableMatcher {
 
-    public static BoundedMatcher<View, ImageView> hasDrawable() {
-        return new BoundedMatcher<View, ImageView>(ImageView.class) {
-            @Override
-            public void describeTo(Description description) {
+    public static Matcher<View> hasDrawable() {
+        return new BoundedDiagnosingMatcher<View, ImageView>(ImageView.class) {
+             @Override
```

Comment:
Super nit, remove the space before `@Override`.

Reply:
Oh sorry, corrected. 
By the way, the documentation of espresso-intents on android developer site (https://developer.android.com/training/testing/espresso/intents) still uses deprecated classes. 
By any chance there's a way to correct it too?

Reply:
Yeah, the documentation should be updated too. Maybe create an issue for updating the doc so that we could track it separately? Thanks!

Reply:
Sure! Thank you! https://github.com/android/android-test/issues/951

Reply:
Teşekkür